src/main/java/io/confluent/kafkarest/ConsumerManager.java

```diff
@@ -145,13 +146,15 @@ public String createConsumer(String group, ConsumerInstanceConfig instanceConfig
     try {
       log.debug("Creating consumer " + name + " in group " + group);
 
+      Properties props;
       // Note the ordering here. We want to allow overrides, but almost all the
       // consumer-specific settings don't make sense to override globally (e.g. group ID, consumer
       // ID), and others we want to ensure get overridden (e.g. consumer.timeout.ms, which we
       // intentionally name differently in our own configs).
-      Properties props = (Properties) config.getOriginalProperties().clone();
-      props.setProperty("zookeeper.connect", zookeeperConnect);
+      props = (Properties) config.getOriginalProperties().clone();
+      props.setProperty("bootstrap.servers", bootstrapServers);
```

Comment:
You can use `KafkaRestConfig.BOOTSTRAP_SERVERS_CONFIG` here instead of `bootstrap.servers`.